File: src/main/java/top/zbeboy/isy/web/vo/graduate/design/release/GraduationDesignReleaseUpdateVo.java
package top.zbeboy.isy.web.vo.graduate.design.release;

import lombok.Data;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by lenovo on 2017-05-06.
 */
@Data
public class GraduationDesignReleaseUpdateVo {
    @NotNull
    @Size(max = 64)
    private String graduationDesignReleaseId;
    @NotNull
    @Size(max = 100)
    private String graduationDesignTitle;
    @NotNull
    private String startTime;
    @NotNull
    private String endTime;
    @NotNull
    private String fillTeacherTime;
    private Byte graduationDesignIsDel;
    private String files;
}

File: src/main/java/top/zbeboy/isy/web/vo/graduate/design/release/GraduationDesignReleaseUpdateVo.kt
package top.zbeboy.isy.web.vo.graduate.design.release

import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2018-01-14 .
 **/
open class GraduationDesignReleaseUpdateVo {
    @NotNull
    @Size(max = 64)
    var graduationDesignReleaseId: String? = null
    @NotNull
    @Size(max = 100)
    var graduationDesignTitle: String? = null
    @NotNull
    var startTime: String? = null
    @NotNull
    var endTime: String? = null
    @NotNull
    var fillTeacherTime: String? = null
    var graduationDesignIsDel: Byte? = null
    var files: String? = null
}